pom.xml

```diff
@@ -440,7 +440,7 @@
     <swagger-parser-version>1.0.6-SNAPSHOT</swagger-parser-version>
     <scala-version>2.10.4</scala-version>
     <felix-version>2.3.4</felix-version>
-    <swagger-core-version>1.5.1-M2</swagger-core-version>
+    <swagger-core-version>1.5.2-M2-SNAPSHOT</swagger-core-version>
```

Comment:
Apologies, I had missed that comment. We can't depend on a SNAPSHOT here though. That means we'll have to wait until swagger-core is released and then modify this PR accordingly. Still not sure what version that's going to end up being.